labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,7 +33,12 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    val input = "ejp mysljylc kd kxveddknmc re jsicpdrysi rbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcd de kr kd eoya kw aej tysr re ujdr lkgc jv zq"
+    val output = "our language is impossible to understand there are twenty six factorial possibilities so it is okay if you want to just give up qz"
+    val mapping = (input zip output).toSet
+    mapping.groupBy(_._1).values.forall(_.size==1)
```

Comment:
I don't think this is necessary. In the previous line, you can convert directly to a map with toMap